src/Extension/ReviewContentNotification.php

```diff
@@ -576,20 +590,36 @@ private function getSecondNotificationDateByArticleId($articleId)
      *
      * @since  1.0.1
      */
-    private function getRecipientsArray($specificEmail, $currentSiteLanguage, $articleObject, $forcedLanguage): array
+    private function getRecipientsArray($specificEmail, $whoEmail, $currentSiteLanguage, $articleObject, $forcedLanguage): array
     {
         $recipients = [];
 
         if (!empty($specificEmail)) {
             $specificEmails = explode(',', $specificEmail);
 
             foreach ($specificEmails as $value) {
-                $recipients[] = ['email' => $value, 'language' => $currentSiteLanguage];
+                if (filter_var($value, FILTER_VALIDATE_EMAIL)) {
+                    $recipients[] = ['email' => $value, 'language' =>  empty($forcedLanguage) ? $currentSiteLanguage : $forcedLanguage];
+                }
             }
         }
 
+        if ($whoEmail == 'none') {
+            if (\count($recipients) && $whoEmail == 'none') {
```

Comment:
```suggestion
            if (\count($recipients)) {
```

whoEmail is checked above right?

Reply:
I have to rework this part. I overlooked the created_by in the loadUserById further below in the code.

Reply:
Ok let me know once done.

Reply:
Ok,
I made the 'Send email to' optional so you can pick which users to send.

I made some changes to prevent duplicate emails and validated the result of loadUserById.


Just for the background; For project with a lot of categories and articles, we want to send emails to one fixed address. Hence this PR and #5.

We also want to send an aggregated list instead of single emails. I solved that issue as wel. But it's a kind of ugly. I can make a PR for that if you like to